scripts/shared/targets.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+function print_indent() {
+    printf "%-24s" "$1"
+    printf "$2"
+}
+
+title=$(print_indent "Target" "Description")
+echo "$title"
+echo "$title" | tr '[:alnum:]' '-'
```

Comment:
Combined with the above:

```suggestion
print_indent Target Description | tee >(tr '[:alnum:]' '-')
```